README.md

```diff
@@ -1,9 +1,73 @@
-# autofs #
+# puppet-module-autofs
+
+Manage autofs
+
 ===
 
-Automounter for NFS.
+# Compatibility
+
+This module has been tested to work on the following systems with Puppet v3 and Ruby versions 1.8.7, 1.9.3 and 2.0.0.
+
+  * RedHat EL5 & EL6
+  * Suse 10 & 11
+  * Ubuntu 12.04
+  * Debian 7.7
+
+===
+
+# Parameters
+
+enable
+------
+Boolean value to enable autofs. 
+
+- *Default*: true
+
+mounttimeout
+-------
+Numeric value to set timeout of mounts in seconds.
+
+- *Default*: 300
+
+umountwait
+----------
+Numeric value to try and prevent expire delays when trying to umount from a server
```

Comment:
What happens if this is undef as opposed to a number? Can this be 0? Can it be negative? If it needs to be positive or non-zero, please note that and add an additional check/test.